sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/PeriodicMetricReaderFactory.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.export;
+
+import io.opentelemetry.sdk.internal.DaemonThreadFactory;
+import java.time.Duration;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+/** Factory for {@link PeriodicMetricReader}. */
+public class PeriodicMetricReaderFactory implements MetricReaderFactory {
```

Comment:
For our general pattern we'd generally make this have a package private constructor and have `PeriodicMetricReader.createFactory()` return it. Assuming we prefer that over static methods on MetricReaderFactory not clear for this one

Reply:
I'm fine with that pattern here.